src/OrchardCore.Modules/OrchardCore.Taxonomies/Drivers/TaxonomyContentsAdminListDisplayDriver.cs

```diff
@@ -100,6 +100,7 @@ public override async Task<IDisplayResult> EditAsync(ContentOptionsViewModel mod
                     })
                     .Location("Actions:40." + position.ToString())
                     .Prefix("Taxonomy" + taxonomy.ContentItemId)
+                    .Differentiator("Taxonomy" + taxonomy.ContentItemId)
```

Comment:
I don't think we should use the `ContentItemId`.

I would suggest the `AliasPart` and normalized as is done in the `MenuShapes`, as I think the alias is on a taxonomy by default. If not Alias, `DisplayText` like the `MenuShapes`.